tensorboard/uploader/uploader.py

```diff
@@ -536,12 +529,7 @@ def _create_run(self, run_name):
             request budget.
         """
         run_proto = self._request.runs.add(name=run_name)
-        # We can't calculate the proto key cost exactly ahead of time, as
```

Comment:
I don't quite understand this and the related comments below.  Should it be retained?  It seems to me that the total cost was being correctly summed up before across levels of the hierarchy, and still is, so you're right to delete the comments... right?

Reply:
I've improved the comments to explain how proto cost is calculated.  Hopefully this is clearer.  Unfortunately I had to familiarize myself with this to convince myself that I could reuse this logic for Tensor sending.